### PR TITLE
FEATURE: Carousel Blocks

### PR DESCRIPTION
**Pull request by unclecheese**
Resolves https://github.com/silverstripeltd/open-sourcerers/issues/101

## Todo

Split out the the blocks functionality and release as OS module.

### CODE REVIEW

**Review by dnsl48 — CHANGES_REQUESTED** [review]
1. needs a rebase
2. "pause" button doesn't seem to work for me.
3. Carousel title doesn't appear anywhere on the page even though I hit the "Displayed" checkbox

File: app/src/Page.php [hunk 1]
```diff
@@ -2,7 +2,13 @@
 
 namespace {
 
+    use DNADesign\Elemental\Extensions\ElementalPageExtension;
```
Comment (dnsl48) [inline app/src/Page.php]: doesn't seem like we need all those imports here?

**Review by dnsl48 — CHANGES_REQUESTED** [review]

File: app/src/Page.php (hunk 1 above)
Comment (unclecheese) [reply app/src/Page.php]: Done

**Comment by unclecheese** [comment]
> "pause" button doesn't seem to work for me.

Make sure you're using the related PR on the theme, as well. :)

**Comment by unclecheese** [comment]
> Carousel title doesn't appear anywhere on the page even though I hit the "Displayed" checkbox

Have added CSS for element titles. Make sure you're on the theme feature branch, as well.

**Review by dnsl48 — APPROVED** [review]
beautiful!
good to merge on green build

**Pull request merged**